plugins/languages/nvim-orgmode.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "orgmode";
+  originalName = "nvim-orgmode"; 
+  defaultPackage = pkgs.vimPlugins.orgmode; 
+
+  maintainers = [ helpers.maintainers.GaspardMonge ]; 
```

```suggestion
  maintainers = [ lib.maintainers.refaelsh ]; 
```

Either this line, or the addition to maintainers.nix, is incorrect.

---

I feel  abit stupid :-) Fixed.